README.md

```diff
@@ -1 +1,65 @@
-# The Odin Project Ruby Exercises
\ No newline at end of file
+# The Odin Project Ruby Exercises
+
+# About this repository
+
+This repository is a complement to the Basic Ruby course for The Odin Project. It contains a series of basic Ruby exercises alongside pre-written tests that will help guide the student towardsa solution, providing a Test Driven Development experience.
```

Comment:
Can we remove "Basic" in the "complement to the Basic Ruby" part here please? We plan on using this repo for exercises for the upcoming new OOP section as well :)

Reply:
Typo on the last sentence - "towardsa"